Trigger/src/ReadTriggerInfo_module.cc

```diff
@@ -286,10 +304,21 @@ namespace mu2e {
     _sdTag         (pset.get<art::InputTag>("strawDigiCollection"  , "makeSD")),
     _chTag         (pset.get<art::InputTag>("comboHitCollection"   , "TTmakeSH")),
     _cdTag         (pset.get<art::InputTag>("caloDigiCollection"   , "CaloDigiFromShower")),
-    _evtWeightTag  (pset.get<art::InputTag>("protonBunchIntensity" , "protonBunchIntensity")),
+    _evtWeightTag  (pset.get<art::InputTag>("protonBunchIntensity" , "PBIsim")),
+    _hsCprTag      (pset.get<art::InputTag>("cprHelixSeedCollection", "CalHelixFinderDe:Positive")), // , "KFFDeMHPar")),
+    _hsTprTag      (pset.get<art::InputTag>("tprHelixSeedCollection", "HelixFinderDe:Positive")), // , "KFFDeMHPar")),
+    _ksTag         (pset.get<art::InputTag>("kalSeedCollection"  , "KFFDeMHPar")),
+    _trkQualTag    (pset.get<art::InputTag>("trackQualCollection", "TrkQualDeMHPar")),
+    _vdTag         (pset.get<art::InputTag>("vdStepPoints","NOTNOW")), // , "compressDigiMCs:virtualdetector")),
     _duty_cycle    (pset.get<float> ("dutyCycle", 1.)),
     _processName   (pset.get<string> ("processName", "globalTrigger")),
-    _nProcess      (pset.get<float> ("nEventsProcessed", 1.))
+    _effBits       (pset.get<std::vector<size_t>>("effBits", std::vector<size_t>{2,8})),
+    _trkMinTanDip  (pset.get<float> ("trkMinTanDip", 0.5)),
```

Comment:
Please  make a sub-table to hold the cuts.  When you make the change, remove the defaults from the code.

Reply:
I can do it, but I'm not super excited about having a table for an analyzer module in the prolog_trigger.fcl file. Are we sure about it?